docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_intro.md

```diff
@@ -0,0 +1,36 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_intro.md"]
+```
+
+# Introduction
+
+Let $\mathbb{F}$ be an ordered field; the most prominent case here is where
```

Comment:
I still don't understand in what sense this is a "prominent" case? Do you mean "the most common" or "most frequently used", or "typically this is..."

Reply:
OK, something weird is going on here: I now see that you answered this question at https://github.com/oscar-system/Oscar.jl/pull/611#discussion_r686700104 but I cannot add to that conversation, nor did it show it to me in the code.. Huh.

Based on what @lkastner wrote there, I'd suggest to basically use his wording from his explanation (i.e., this is most frequently used, and for other fields not all features may be supported -- although of course ideally we'd be more specific as to what works and what not, but that can be in a future PR)

Reply:
While I am not picky about the wording, we need to compromise a bit here: Currently we only support rational polytopes etc in Oscar, but polymake can do much more.  The relevant ordered fields (e.g., rational functions) exist in Flint etc, but we need some glue code (which is in the making).  Therefore I'd prefer this slightly more general description to sketch our intentions.